Solutions/OracleWebLogicServer/Analytic Rules/OracleWebLogicMultipleServerErrorsRequestsFromSingleIP.yaml

```diff
@@ -0,0 +1,28 @@
+id: 268f4fde-5740-11ec-bf63-0242ac130002
+name: Oracle - Multiple server errors from single IP
+description: |
+  'Detects multiple server errors from one source in short timeframe'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: OracleWebLogicServer
+    dataTypes:
+      - OracleWebLogicServerEvent

```

Comment:
Please add queryFrequency,queryPeriod,triggerOperator and triggerThreshold

Reply:
fixed